auth/core/multiauth.go

```diff
@@ -13,10 +13,7 @@ type MultiAuth struct {
 }
 
 func (m *MultiAuth) Add(a Auth) {
-	if m.auths == nil {
-		m.auths = make([]Auth, 0)
```

Comment:
do we ever need to run make to initialize this?

Reply:
append does this for us. if m.auths is nil, append allocates the underlying memory